src/Select/index.js

```diff
@@ -61,7 +61,7 @@ const Select = React.forwardRef(
             )}
             {...rest}
           >
-            <option value="" disabled selected>
+            <option value="" disabled>
```

Comment:
shouldn't it be `defaultSelected` as the warning seems to indicate?

Reply:
`defaultValue` or `value` prop into <select /> element instead of `<option />`